### PR TITLE
Add flag to remove unused variables, add Combined Bayesian PID with TPC only

### PR DESCRIPTION
**Pull request by AnnalenaKa**
A flagg is added to the task to switch off the storing of variables in the tree that are not used in the current analyses. Furthermore a new variable is added which is the combined bayesian PID using only the TPC and no TOF information.

### CODE REVIEW

**Review by fgrosa — COMMENTED** [review]
Hi @AnnalenaKa thanks for the implementation! There is a small detail, when fixed I will approve it. Cheers, Fabrizio

File: PWGHF/vertexingHF/AliAnalysisTaskSELc2pKs0fromKFP.h [hunk 1]
```diff
@@ -147,6 +148,7 @@ class AliAnalysisTaskSELc2pKs0fromKFP : public AliAnalysisTaskSE
         Bool_t                  fIsMC; ///< Flag of MC analysis
         Bool_t                  fUseWeights; ///< Flag to use pT weight functions
         Bool_t                  fKeepOnlyMCSignal; ///< flag to keep only signal candidates
+        Bool_t                  fKeepAllVariables; ///<flag to keep all possible variables that were removed to reduce the tree size
```
Comment (fgrosa) [inline PWGHF/vertexingHF/AliAnalysisTaskSELc2pKs0fromKFP.h:151]: Every time you add a data member you should update the `ClassDef` (in this case from `7` to `8`)

Comment (AnnalenaKa) [reply PWGHF/vertexingHF/AliAnalysisTaskSELc2pKs0fromKFP.h:151]: Thanks a lot for spotting this. I did not see this.

**Comment by fgrosa** [comment]
+1

**Pull request merged**